listenbrainz/spark/test_request_manage.py

```diff
@@ -27,49 +27,56 @@
 from click.testing import CliRunner
 from listenbrainz.spark import request_manage
 
+
```

Comment:
Sorry for the extra PEP8 changes, this file had too many and bot kept complaining